ui/src/views/Quickstart/index.jsx

```diff
@@ -378,41 +396,49 @@ export default class QuickStart extends Component {
           </HelpView>
         }>
         <Fragment>
-          <div className={classes.orgRepo}>
-            <TextField
-              label="Organization Name"
-              name="owner"
-              onChange={this.handleOrgRepoChange}
-              value={owner}
-              autoFocus
-            />
-            <Typography className={classes.separator} variant="h5">
-              /
-            </Typography>
-            <TextField
-              label="Repository Name"
-              name="repo"
-              onChange={this.handleOrgRepoChange}
-              value={repo}
-            />
-            <div className={classes.iconContainer}>
-              {(installedState === 'success' && (
-                <CheckIcon className={classes.checkIcon} />
+          <List>
+            <ListItem>
+              <TextField
+                label="Organization Name"
+                name="owner"
+                fullWidth
+                onChange={this.handleOrgRepoChange}
+                value={owner}
+                autoFocus
+              />
+              <Typography className={classes.separator} variant="h5">
+                /
+              </Typography>
+              <TextField
+                label="Repository Name"
+                name="repo"
+                fullWidth
+                onChange={this.handleOrgRepoChange}
+                value={repo}
+              />
+            </ListItem>
+          </List>
+
+          <div className={classes.iconContainer}>
+            {(installedState === 'success' && (
+              <CheckIcon className={classes.checkIcon} />
+            )) ||
+              (installedState === 'error' && (
+                <AlertCircleOutlineIcon className={classes.errorIcon} />
               )) ||
-                (installedState === 'error' && (
-                  <AlertCircleOutlineIcon className={classes.errorIcon} />
-                )) ||
-                (installedState === 'loading' && <Spinner size={24} />)}
-            </div>
+              (installedState === 'loading' && <Spinner size={24} />)}
           </div>
+
```

Comment:
![Screen Shot 2019-10-18 at 11 36 01 AM](https://user-images.githubusercontent.com/3766511/67108053-a7dcc880-f19b-11e9-96c0-93bda9d382cd.png)

The exclamation mark icon looks out out place. I think it used to sit beside the repository name.